reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -0,0 +1,1029 @@
+          "KeyName" : {"Ref": "KeyName"},
+	      "InstanceType": {"Ref": "MasterInstanceType"},
+          "SecurityGroupIds": [{ "Fn::GetAtt" : ["NodeSG", "GroupId"] }, { "Fn::GetAtt" : ["MasterSG", "GroupId"] }, { "Fn::GetAtt" : ["EtcdSG", "GroupId"] }],
+          "SubnetId" : {"Ref": "PrivateSubnet2"},
+          "IamInstanceProfile": { "Ref": "MasterInstanceProfile" },
+          "Tags": [
+            { "Key": "Name",
+              "Value": {"Fn::Join": [".", ["ose-master02",{"Ref": "PublicHostedZone"}]]}
+            },
+            { "Key": "openshift-role",
+              "Value": "master"
+            }
+          ],
+          "BlockDeviceMappings" : [
+          {
+            "DeviceName": "/dev/sda1",
+            "Ebs": {
+              "DeleteOnTermination": "true",
+              "VolumeSize": {"Ref": "MasterRootVolSize"},
+              "VolumeType": {"Ref": "MasterRootVolType"}
+            }
+          },
+          {
+            "DeviceName": "/dev/xvdb",
+            "Ebs": {
+              "DeleteOnTermination": "true",
+              "VolumeSize": {"Ref": "MasterDockerVolSize"},
+              "VolumeType": {"Ref": "MasterDockerVolType"}
+            }
+          },
+          {
+            "DeviceName": "/dev/xvdc",
+            "Ebs": {
+              "DeleteOnTermination": "true",
+              "VolumeSize": {"Ref": "MasterEtcdVolSize"},
+              "VolumeType": {"Ref": "MasterEtcdVolType"}
+            }
+          }
+         ]
+     }
+   },
+    "Master03" : {
+       "Type" : "AWS::EC2::Instance",
+       "Properties" : {
+          "ImageId" : {"Ref": "AmiId"},
+          "KeyName" : {"Ref": "KeyName"},
+	      "InstanceType": {"Ref": "MasterInstanceType"},
+          "SecurityGroupIds": [{ "Fn::GetAtt" : ["NodeSG", "GroupId"] }, { "Fn::GetAtt" : ["MasterSG", "GroupId"] }, { "Fn::GetAtt" : ["EtcdSG", "GroupId"] }],
+          "SubnetId" : {"Ref": "PrivateSubnet3"},
+          "IamInstanceProfile": { "Ref": "MasterInstanceProfile" },
+          "Tags": [
+            { "Key": "Name",
+              "Value": {"Fn::Join": [".", ["ose-master03",{"Ref": "PublicHostedZone"}]]}
+            },
+            { "Key": "openshift-role",
+              "Value": "master"
+            }
+          ],
+          "BlockDeviceMappings" : [
+          {
+            "DeviceName": "/dev/sda1",
+            "Ebs": {
+              "DeleteOnTermination": "true",
+              "VolumeSize": {"Ref": "MasterRootVolSize"},
+              "VolumeType": {"Ref": "MasterRootVolType"}
+            }
+          },
+          {
+            "DeviceName": "/dev/xvdb",
+            "Ebs": {
+              "DeleteOnTermination": "true",
+              "VolumeSize": {"Ref": "MasterDockerVolSize"},
+              "VolumeType": {"Ref": "MasterDockerVolType"}
+            }
+          },
+          {
+            "DeviceName": "/dev/xvdc",
+            "Ebs": {
+              "DeleteOnTermination": "true",
```

Comment:
Since this volume houses the etcd data, it may make sense to set DeleteOnTermination to false to prevent the data volume from disappearing accidentally..  Cleanup would require a second step to remove the volumes, though.